### PR TITLE
Trimesh is a shapely-like library used by some of our users

### PR DESCRIPTION
**Pull request by surak**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by surak** [comment]
I don't get the error. So, PythonPackage defaults to Pipy_source, that's it?

File: easybuild/easyconfigs/t/trimesh/trimesh-3.17.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['025bb2fa3a2e87bdd6873f11db45a7ca19216f2f8b6aed29140fca57e32c298e']
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
+
+
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/t/trimesh/trimesh-3.17.1-foss-2022a.eb:23]: The CI check does not like to 3 empty lines. Maybe you can remove one.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bwd-rockylinux-91 - Linux Rocky Linux 9.1 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 515.86.01, Python 3.9.14
See https://gist.github.com/cda21431578857ee572e136996eb4e3c for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @surak!

**Pull request merged**